frontend/src/container/GridGraphLayout/Graph/FullView/index.tsx

```diff
@@ -202,6 +203,12 @@ const FullView = ({
 					name,
 				}}
 			/>
+
+			{state.error &&
```

Comment:
instead of using here

can we use inside of the useEffect ?

Reply:
No, I don't think we can place it inside useEffect because If we click on refresh button then 'useEffect' is not called after that. So, if some error has occurred in that call then it would not show notification.

Reply:
we can listen to the decencies of the `useEffect` in `useEffectCallback` 

current approach might trigger multiple render of the notification

Reply:
@pal-sig Yes, I noticed this issue recently and changed the code to resolve it